ledger/acctupdates.go

```diff
@@ -1208,6 +1212,75 @@ func (au *accountUpdates) lookupResource(rnd basics.Round, addr basics.Address,
 	}
 }
 
+func (au *accountUpdates) lookupAssetResources(rnd basics.Round, addr basics.Address, assetIDGT basics.AssetIndex, limit uint64, synchronized bool) (data []ledgercore.AssetResourceWithIDs, validThrough basics.Round, err error) {
```

Comment:
My gut says that all of the synchronization and double checking of db can go away if we're comfortable with ignoring changes in the deltas.

I do think such an API is very useful, but I wonder if we should decide that this entire "class" of APIs should be identified somehow. Like calling them "untimely" or something.

Reply:
So we are not returning any information from the deltas atm - I wanted to avoid the complexity there. The idea being suggested is to return both round values in the API, but okay with the resources being sourced solely from database?

Reply:
Yes, I think that there's value in these "untimely" APIs (not the least of which is ease of implementation), I just think we need to be very clear about what they are.  That's why I'd like something in the name, and to report the latest round number too.

Reply:
Latest round number is reported from the db - I did not special name the APIs.